Review

Rashid-creator-droid opened the pull request.

kmvpw commented on eatpoint/api/serializers/reservations.py:
```diff
@@ -0,0 +1,198 @@
+from reservation.models import Reservation, ReservationHistory, Availability
+
+
+# class AuthReservationsEditSerializer(serializers.ModelSerializer):
```
Нет пояснения - что в этом коде, почему он остался закомментированным, что с нем делать.

kmvpw commented on eatpoint/api/serializers/users.py:
```diff
@@ -0,0 +1,226 @@
+            raise serializers.ValidationError(
+                f"Роль должна быть {core.constants.CLIENT} или {core.constants.RESTORATEUR}"
+            )
+        if (user_with_email.exists() or user_with_telephone.exists()) and (
```
Какая-то сложная проверка. А если выше выполнить поиск пользователя по полю email и is_active?

kmvpw commented on eatpoint/api/serializers/users.py:
```diff
@@ -0,0 +1,226 @@
+            .exclude(id=user.id)
+            .exists()
+        ):
+            raise ValidationError("Email уже зарезервирован!")
```
Текст исключения не соответствует проверяемому полю

kmvpw commented on eatpoint/api/views/code_generate.py:
```diff
@@ -0,0 +1,101 @@
+                {"detail": "Номер телефона подтвержден!"},
+                status=status.HTTP_200_OK,
+            )
+        else:
```
Почему решили отдельный код написать если не прошла валидация (не восиользовались raise exception)?

kmvpw commented on eatpoint/api/views/establishments.py:
```diff
@@ -0,0 +1,392 @@
+        removable = Favorite.objects.filter(
+            user=user, establishment=establishment
+        )
+        if not removable.exists():
```
Проверка с not - плохая практика. Лучше переписать без использования not.

kmvpw commented on eatpoint/core/constants.py:
```diff
@@ -0,0 +1,62 @@
+INTERVAL_MINUTES = 30
+
+
+START_TIME = "00:00"
```
Некоторые константы подписаны, некоторые - нет. Что означает START_TIME?

kmvpw reviewed (commented):
Нет отписания PR - что было сделано, особенности, на что обратить внимение
По тестам - предлагаю настроить автозапуск тестов после любых изменений в репозитории.
Поясню - при проведении ревью можно быть уверенным (посмотрев на выполненные тесты), что 
код соответствует стандарту, решение запускается, базовые проверки выполнены.

Названия коммитов (не всех) не отражает суть слияния.

Rashid-creator-droid committed:
Merge pull request #233 from finding-and-booking-tables-restaurants/fix/filter

фильтрация по имени

Rashid-creator-droid committed:
Merge pull request #234 from finding-and-booking-tables-restaurants/alpha_v1

Alpha v1

Rashid-creator-droid committed:
админка картинок

Rashid-creator-droid committed:
Merge pull request #236 from finding-and-booking-tables-restaurants/fix/image_admin

админка картинок

Rashid-creator-droid committed:
фикс картинок

Rashid-creator-droid committed:
Merge pull request #237 from finding-and-booking-tables-restaurants/fix/image_admin

фикс картинок

Rashid-creator-droid committed:
фикс картинок

Rashid-creator-droid committed:
Merge pull request #238 from finding-and-booking-tables-restaurants/fix/image_admin

фикс картинок

Rashid-creator-droid committed:
фикс постера

Rashid-creator-droid committed:
Merge pull request #239 from finding-and-booking-tables-restaurants/fix/image_admin

фикс постера

Rashid-creator-droid committed:
Merge pull request #240 from finding-and-booking-tables-restaurants/alpha_v1

Alpha v1

SemenovY committed:
обновил models

Rashid-creator-droid committed:
Эндпоинт длявывода времени и даты

Rashid-creator-droid committed:
Эндпоинт длявывода времени и даты

Rashid-creator-droid committed:
Merge pull request #242 from finding-and-booking-tables-restaurants/dev/time_date_availability

Dev/time date availability

SemenovY committed:
убрал лишние коментарии

убрал в юзерс дубликат extra_kwargs

Rashid-creator-droid committed:
fix test

Rashid-creator-droid committed:
Merge pull request #243 from finding-and-booking-tables-restaurants/dev/time_date_availability

fix test

SemenovY committed:
pep8 fix

pre-commit

Rashid-creator-droid committed:
Merge pull request #241 from finding-and-booking-tables-restaurants/dev/owner_response

Отзывы на заведение и ответ владельца на отзыв

Rashid-creator-droid committed:
Merge pull request #244 from finding-and-booking-tables-restaurants/alpha_v1

Alpha v1

Rashid-creator-droid committed:
fix status reservation

Rashid-creator-droid committed:
Merge pull request #246 from finding-and-booking-tables-restaurants/fix/reservation-status

Fix/reservation status

Rashid-creator-droid committed:
Merge pull request #247 from finding-and-booking-tables-restaurants/develop

Develop

SemenovY committed:
убрал поля review и owner из запроса

оставлен только текст

SemenovY committed:
ответ владельца заведения привязан к соответствующему отзыву

в отзыве будет содержаться поле response, которое будет содержать информацию о ответе владельца заведения на этот отзыв

SemenovY committed:
update permissions review

SemenovY committed:
создано отдельное приложение отзывы

обновлены все связи, сериализатор и вью

Rashid-creator-droid committed:
Merge pull request #248 from finding-and-booking-tables-restaurants/dev/owner_response

fix убраны поля review и owner, привязан ответ к отзыву

Rashid-creator-droid committed:
Merge pull request #249 from finding-and-booking-tables-restaurants/dev/review

fix: Создано приложение reviews

SemenovY committed:
add migrations

0007_delete_review, 0001_inital

Rashid-creator-droid committed:
Merge pull request #250 from finding-and-booking-tables-restaurants/dev/review

Dev/review add migrations

Rashid-creator-droid committed:
deploy

Rashid-creator-droid committed:
Merge pull request #253 from finding-and-booking-tables-restaurants/fix/reservation-status

deploy

Rashid-creator-droid committed:
Merge pull request #254 from finding-and-booking-tables-restaurants/develop

Develop

Rashid-creator-droid committed:
deploy

Rashid-creator-droid committed:
Merge pull request #255 from finding-and-booking-tables-restaurants/fix/reservation-status

deploy

Rashid-creator-droid committed:
Merge pull request #256 from finding-and-booking-tables-restaurants/alpha_v1

Alpha v1

ChernyaevAV committed:
fix: исправил валидацию пароля + разрешил отправку email при регистрации

Rashid-creator-droid committed:
Merge pull request #257 from finding-and-booking-tables-restaurants/fix/password_validate

fix: исправил валидацию пароля + разрешил отправку email при регистрации

ChernyaevAV committed:
fix: исправил EMAIL_USE_SSL и EMAIL_USE_TLS

Rashid-creator-droid committed:
Merge pull request #260 from finding-and-booking-tables-restaurants/fix/settings_for_send_email

fix: исправил EMAIL_USE_SSL и EMAIL_USE_TLS

SemenovY committed:
refactor serializers and views analytics

add Count and replace Reservation -> ReservationHistory

ChernyaevAV committed:
fix: теперь можно отправлять EMAIL + добавил новый домен eatpoint.site

Rashid-creator-droid committed:
Merge pull request #262 from finding-and-booking-tables-restaurants/fix/settings_for_send_email

fix: теперь можно отправлять EMAIL + добавил новый домен eatpoint.site

ChernyaevAV committed:
fix: добавил ёЁ в валидацию ФИО пользователя

SemenovY committed:
get add permissions

SemenovY committed:
add days

Rashid-creator-droid committed:
Merge pull request #265 from finding-and-booking-tables-restaurants/fix/validate_fio

fix: добавил ёЁ в валидацию ФИО пользователя

SemenovY committed:
analytics for past reserv have been completed

SemenovY committed:
add analytics reservation

Rashid-creator-droid committed:
Merge pull request #267 from finding-and-booking-tables-restaurants/dev/analytics

Dev/analytics

Rashid-creator-droid committed:
200 destroy

Rashid-creator-droid committed:
Merge pull request #268 from finding-and-booking-tables-restaurants/dev/status_code

200 destroy

ChernyaevAV committed:
fix: исправил статус ответа HTTP_409_CONFLICT если при регистрации пользователь уже есть в базе и активный и не активный

ChernyaevAV committed:
fix: поменял метод подтверждения кода на емайл вместо телефона

Rashid-creator-droid committed:
Merge pull request #271 from finding-and-booking-tables-restaurants/fix/confirm_code_from_email

fix: поменял метод подтверждения кода на емайл вместо телефона

Rashid-creator-droid committed:
Merge pull request #270 from finding-and-booking-tables-restaurants/fix/status409_if_no_unique_user

fix: исправил статус ответа HTTP_409_CONFLICT если при регистрации по…

ratarov committed:
made separate app for events

ChernyaevAV committed:
feat: v2 новая модель Бронирования

ChernyaevAV committed:
feat: v2 новая модель Table

ChernyaevAV committed:
feat: v2 доп валидация для анонима и константа возможных дней предварительного бронирования

ChernyaevAV committed:
feat: v2 api

ChernyaevAV committed:
feat: v2 api

ratarov committed:
added events api, without photos

ratarov committed:
added event photo api

ChernyaevAV committed:
feat: v2 api

ChernyaevAV committed:
feat: исправил текст сообщения о подтверждении брони

ChernyaevAV committed:
feat: AVAILABLE_DAYS = 30

Rashid-creator-droid committed:
Merge pull request #274 from finding-and-booking-tables-restaurants/dev/new_reservation

Dev/new reservation

ratarov committed:
fixed events serializers and schema

ratarov committed:
added events validation by est+name+date

ratarov committed:
fix folders

ratarov committed:
added vscode folder to gitignore

ratarov committed:
fixed fixtures

Rashid-creator-droid committed:
Merge branch 'develop' into dev/events

Rashid-creator-droid committed:
Merge pull request #276 from finding-and-booking-tables-restaurants/dev/events

Dev/events

ChernyaevAV committed:
feat: добавил миграции для Бронирований и Заведений, удалил из v2 "события"

ChernyaevAV committed:
fix: удалил настройки для старого домена

Rashid-creator-droid committed:
Merge pull request #277 from finding-and-booking-tables-restaurants/fix/migrate_new_models

feat: добавил миграции для Бронирований и Заведений,

ChernyaevAV committed:
fix: новые настройки для воркфлоу

ChernyaevAV committed:
Merge pull request #282 from finding-and-booking-tables-restaurants/dev/new_settings_workflow

Dev/new settings workflow

ChernyaevAV committed:
Update models.py

ChernyaevAV committed:
fix: новые настройки для воркфлоу

ChernyaevAV committed:
Merge pull request #283 from finding-and-booking-tables-restaurants/dev/new_settings_workflow

fix: новые настройки для воркфлоу

ChernyaevAV committed:
fix: новые настройки для воркфлоу

ChernyaevAV committed:
Update eatpoint_workflows.yml

ChernyaevAV committed:
Merge pull request #284 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update eatpoint_workflows.yml

ChernyaevAV committed:
fix: новые настройки для воркфлоу

ChernyaevAV committed:
fix: изменил имя образа dockerhub

ChernyaevAV committed:
Merge pull request #285 from finding-and-booking-tables-restaurants/dev/new_settings_workflow

Dev/new settings workflow

ratarov committed:
events: removed date_end field, changed qs with list of events to future events

ratarov committed:
added create event seria services

ratarov committed:
fixed events models for migrations

ChernyaevAV committed:
dev: отключил urls v1

ChernyaevAV committed:
Merge pull request #286 from finding-and-booking-tables-restaurants/dev/transfer_to_v2

dev: отключил urls v1

ChernyaevAV committed:
Merge branch 'main' into develop

ChernyaevAV committed:
Merge pull request #287 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: добавил условия для подтверждения брони и для отметки о посещении заведения

ChernyaevAV committed:
Merge pull request #288 from finding-and-booking-tables-restaurants/fix/reservation_accepted_and_visited

fix: добавил условия для подтверждения брони и для отметки о посещении

ChernyaevAV committed:
Merge pull request #289 from finding-and-booking-tables-restaurants/develop

fix: добавил условия для подтверждения брони и для отметки о посещении

ChernyaevAV committed:
dev: celery для периодических напоминаний и

ratarov committed:
events: creation of events seria is working

ratarov committed:
Merge branch 'dev/events' of github.com:finding-and-booking-tables-restaurants/eatpoint-backend into dev/events

ChernyaevAV committed:
fix: исправил условия для отправки напоминания ресторану о подтверждении брони (каждую минуту до начала брони)

ratarov committed:
added api v2 structure

ratarov committed:
Merge branch 'develop' into dev/events

ratarov committed:
events: fixed cover_image, urls, update event service

ratarov committed:
events: added update of events seria

ratarov committed:
added events seria deletion, pagination

ratarov committed:
events: added exceptions and handling

ChernyaevAV committed:
fix: добавил условия для удаления брони

ChernyaevAV committed:
fix: добавил фильтр по количеству мест для слотов

ChernyaevAV committed:
Merge pull request #291 from finding-and-booking-tables-restaurants/dev/filtres_for_slots

fix: добавил фильтр по количеству мест для слотов

ChernyaevAV committed:
Merge pull request #292 from finding-and-booking-tables-restaurants/develop

Develop

ratarov committed:
events: fixed update of events seria

ratarov committed:
events: updated schema

ratarov committed:
events: added bad request for delete seria if nothing was deleted

ratarov committed:
merged migrations

ratarov committed:
events: fixed migrations

ChernyaevAV committed:
Merge pull request #293 from finding-and-booking-tables-restaurants/dev/events

Dev/events

ChernyaevAV committed:
Merge pull request #294 from finding-and-booking-tables-restaurants/develop

Dev/events

ChernyaevAV committed:
Update services.py

ChernyaevAV committed:
Merge pull request #295 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update services.py

ChernyaevAV committed:
Merge pull request #296 from finding-and-booking-tables-restaurants/develop

Dev/events

ChernyaevAV committed:
Update services.py

ChernyaevAV committed:
Merge pull request #297 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update services.py

ChernyaevAV committed:
Merge pull request #298 from finding-and-booking-tables-restaurants/develop

Dev/events

ChernyaevAV committed:
Update eatpoint_workflows.yml

ChernyaevAV committed:
Merge pull request #299 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update eatpoint_workflows.yml

ChernyaevAV committed:
Update eatpoint_workflows.yml

Добавил вход в докерхаб перед скачивание образа, т.к. сделал приватный репозиторий

ChernyaevAV committed:
Merge pull request #300 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update eatpoint_workflows.yml

ChernyaevAV committed:
fix: перенес теги схемы во вьюхи

ChernyaevAV committed:
dev: добавил ежедневные задачи по удалению старых слотов и по созданию слотов

ChernyaevAV committed:
dev: добавил задачи по копированию брони в архив и удалению брони после визита

ChernyaevAV committed:
dev: добавил задачи по копированию брони в архив и удалению брони после визита

ChernyaevAV committed:
рефакторинг

ChernyaevAV committed:
fix: добавил отображение брони устаревшей брони (если слоты удалены)

ChernyaevAV committed:
Merge pull request #290 from finding-and-booking-tables-restaurants/dev/celery_periodic_reminds

dev: celery для периодических напоминаний

ChernyaevAV committed:
Merge pull request #301 from finding-and-booking-tables-restaurants/develop

Dev: celery for periodic tasks

ChernyaevAV committed:
fix: исправил имена задач

ChernyaevAV committed:
Merge pull request #302 from finding-and-booking-tables-restaurants/fix/rename_tasks

fix: исправил имена задач

ChernyaevAV committed:
Merge pull request #303 from finding-and-booking-tables-restaurants/develop

fix: исправил имена задач

ChernyaevAV committed:
fix: подключил фронт

ChernyaevAV committed:
Merge pull request #304 from finding-and-booking-tables-restaurants/fix/compose_for_frontend

fix: подключил фронт

ChernyaevAV committed:
Merge pull request #305 from finding-and-booking-tables-restaurants/develop

fix: подключил фронт

ChernyaevAV committed:
fix: исправил задачу по копированию в архив и удалению исполненных броней, в модель Истории добавил поле id_брони.

ChernyaevAV committed:
Merge pull request #306 from finding-and-booking-tables-restaurants/fix/celery_delete_reservations

fix: исправил задачу по копированию в архив и удалению исполненных бр…

ChernyaevAV committed:
Merge pull request #307 from finding-and-booking-tables-restaurants/develop

fix: исправил задачу по копированию в архив и удалению исполненных броней

ChernyaevAV committed:
Update Dockerfile

ChernyaevAV committed:
Merge pull request #308 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update Dockerfile

ChernyaevAV committed:
fix: таймаут=120 и воркеры=3 для gunicorn + рефакторинг

ChernyaevAV committed:
Merge pull request #309 from finding-and-booking-tables-restaurants/fix/reservations

fix: таймаут=120 и воркеры=3 для gunicorn + рефакторинг

ChernyaevAV committed:
Update README.md

ChernyaevAV committed:
Merge pull request #310 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Chernyaev av patch 1

ChernyaevAV committed:
fix: исправил сообщения при изменении статуса брони

ChernyaevAV committed:
Merge pull request #311 from finding-and-booking-tables-restaurants/fix/set_accepted_visited

Fix/set accepted visited

ChernyaevAV committed:
Merge pull request #312 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: CRUD для брони (Ресторатор)

ChernyaevAV committed:
Merge branch 'develop' into fix/CRUD_reservation

ChernyaevAV committed:
Merge pull request #313 from finding-and-booking-tables-restaurants/fix/CRUD_reservation

Fix/crud reservation

ChernyaevAV committed:
Merge pull request #314 from finding-and-booking-tables-restaurants/develop

fix: CRUD reservation

ChernyaevAV committed:
fix: изменил формат даты на iso-8601 (по умолчанию)

ChernyaevAV committed:
Merge pull request #315 from finding-and-booking-tables-restaurants/fix/dateformat_settings

fix: изменил формат даты на iso-8601 (по умолчанию)

ChernyaevAV committed:
fix: добавил примеры для запроса и ответа бронирования для анонима и пользователя

ChernyaevAV committed:
Merge pull request #316 from finding-and-booking-tables-restaurants/fix/dateformat_settings

fix: добавил примеры для запроса и ответа бронирования для анонима и пользователя

ChernyaevAV committed:
Merge pull request #317 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: изменил методы создания брони для зарег и незарег пользователей

ChernyaevAV committed:
Merge pull request #318 from finding-and-booking-tables-restaurants/fix/dateformat_settings

fix: изменил методы создания брони для зарег и незарег пользователей

ChernyaevAV committed:
Merge pull request #319 from finding-and-booking-tables-restaurants/fix/dateformat_settings

fix: изменил методы создания брони для зарег и незарег пользователей

ChernyaevAV committed:
fix: изменил методы создания брони для зарег и незарег пользователей

ChernyaevAV committed:
Merge pull request #320 from finding-and-booking-tables-restaurants/fix/create_reservation

fix: изменил методы создания брони для зарег и незарег пользователей

ChernyaevAV committed:
Merge branch 'main' into develop

ChernyaevAV committed:
Merge pull request #321 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: изменил методы создания брони для зарег и незарег пользователей

ChernyaevAV committed:
Merge pull request #322 from finding-and-booking-tables-restaurants/fix/create_reservation

fix: изменил методы создания брони для зарег и незарег пользователей

ChernyaevAV committed:
fix: изменил отображение полей Слотов в админке. Должна увеличиться скорость обработки страницы в админке

ChernyaevAV committed:
Merge pull request #324 from finding-and-booking-tables-restaurants/fix/optimization_reservation

fix: изменил отображение полей Слотов в админке.

ChernyaevAV committed:
Merge pull request #325 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: изменил фильтрацию броней по емайл ресторана, а не владельца.

ChernyaevAV committed:
Merge pull request #326 from finding-and-booking-tables-restaurants/fix/business_reservation

fix: изменил фильтрацию броней по емайл ресторана, а не владельца.

ChernyaevAV committed:
Merge pull request #327 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: изменил форму ответа на запрос бронирований для клиента и для бизнеса

ChernyaevAV committed:
Merge pull request #328 from finding-and-booking-tables-restaurants/fix/response_for_reservation

fix: изменил форму ответа на запрос бронирований

ChernyaevAV committed:
Merge branch 'main' into develop

ChernyaevAV committed:
Merge pull request #329 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: контакты зарегистрированного пользователя тоже заносятся в форму брони

ChernyaevAV committed:
Merge pull request #330 from finding-and-booking-tables-restaurants/fix/response_resevation

fix: контакты зарегистрированного пользователя тоже заносятся в форму…

ChernyaevAV committed:
Merge pull request #331 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
dev: добавил к модели Бронирования поле is_deleted для возможности отмены

ChernyaevAV committed:
fix: добавил возвращения исключения в случае отправки иного метода кроме is_deleted для клиента

ChernyaevAV committed:
Merge pull request #332 from finding-and-booking-tables-restaurants/dev/reservation_is_deleted

Dev/reservation is deleted

ChernyaevAV committed:
Merge pull request #333 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: исправил ошибку при архивировании броней

ChernyaevAV committed:
Merge pull request #334 from finding-and-booking-tables-restaurants/dev/reservation_is_deleted

fix: исправил ошибку при архивировании броней

ChernyaevAV committed:
Merge pull request #335 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: подтвержденную бронь до срока может удалить только администратор заведения

ChernyaevAV committed:
fix: подтвержденную бронь до срока может удалить только администратор заведения + исправил ошибку обновления статуса слота при отмене и удалении брони

ChernyaevAV committed:
Merge pull request #336 from finding-and-booking-tables-restaurants/fix/delete_resevations

Fix/delete resevations

ChernyaevAV committed:
Merge pull request #337 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: для отмененных броней не отправляются письма админу ресторана и удаляются каждое 1 число

ChernyaevAV committed:
Merge pull request #338 from finding-and-booking-tables-restaurants/fix/send_email_for_accept

fix: для отмененных броней

ChernyaevAV committed:
Merge pull request #339 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: оптимизация запросов для Establishments

ChernyaevAV committed:
fix: оптимизация запросов для Slots

ChernyaevAV committed:
fix: оптимизация запросов для Establishment

ChernyaevAV committed:
Merge pull request #340 from finding-and-booking-tables-restaurants/fix/optimizing_requests

Fix/optimizing requests

ChernyaevAV committed:
Merge pull request #341 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: оптимизация запросов для Отзывов заведений

ChernyaevAV committed:
Merge pull request #342 from finding-and-booking-tables-restaurants/fix/optimizing_requests

fix: оптимизация запросов для Отзывов заведений

ChernyaevAV committed:
Merge pull request #343 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
fix: изменил колчиство выдаваемыз Слотов на странице (по 100)

ChernyaevAV committed:
Merge pull request #344 from finding-and-booking-tables-restaurants/fix/slots_per_page

fix: Слотов на странице (по 100)

ChernyaevAV committed:
Merge pull request #345 from finding-and-booking-tables-restaurants/develop

Слотов на странице (по 100)

ChernyaevAV committed:
fix: добавил фильтры для Слотов по ['date', 'seats', 'zone']

ChernyaevAV committed:
fix: добавил фильтры для Слотов по ['date', 'seats', 'zone']

ChernyaevAV committed:
fix: добавил фильтры для Слотов по ['date', 'seats', 'zone']

ChernyaevAV committed:
Merge pull request #346 from finding-and-booking-tables-restaurants/fix/slots_per_page

фильтры для Слотов

ChernyaevAV committed:
Merge pull request #347 from finding-and-booking-tables-restaurants/develop

фильтры для Слотов

ChernyaevAV committed:
fix: исправил ошибки запроса Заведений для бизнеса

ChernyaevAV committed:
Merge pull request #349 from finding-and-booking-tables-restaurants/fix/slots_per_page

fix: исправил ошибки запроса Заведений для бизнеса

ChernyaevAV committed:
Merge pull request #350 from finding-and-booking-tables-restaurants/develop

fix: исправил ответ на запрос Заведений для бизнеса

ChernyaevAV committed:
fix: исправил выборку Избранного

ChernyaevAV committed:
Merge pull request #351 from finding-and-booking-tables-restaurants/fix/get_favorited

fix: исправил выборку Избранного

ChernyaevAV committed:
Merge pull request #352 from finding-and-booking-tables-restaurants/develop

fix: исправил выборку Избранного

ChernyaevAV committed:
Update README.md

ChernyaevAV committed:
Merge pull request #353 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update README.md

ChernyaevAV committed:
fix: исправлен синтаксис выборки слотов

ChernyaevAV committed:
Merge pull request #354 from finding-and-booking-tables-restaurants/fix/get_slots

fix: исправлен синтаксис выборки слотов

ChernyaevAV committed:
Merge pull request #355 from finding-and-booking-tables-restaurants/develop

fix: исправлен синтаксис выборки слотов

ChernyaevAV committed:
Update reservation.py

добавил id в кверисет слотов

ChernyaevAV committed:
Merge pull request #356 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update reservation.py

ChernyaevAV committed:
Update reservation.py

ChernyaevAV committed:
Merge pull request #357 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

"id" for Response Available Slots

ChernyaevAV committed:
Update reservation.py

ChernyaevAV committed:
Merge pull request #358 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update reservation.py

ChernyaevAV committed:
fix: оптимизация запросов на Бронирования и Слоты

ChernyaevAV committed:
Merge pull request #359 from finding-and-booking-tables-restaurants/fix/optimization_requests

fix: оптимизация запросов на Бронирования и Слоты

ChernyaevAV committed:
Merge branch 'main' into develop

ChernyaevAV committed:
Merge pull request #360 from finding-and-booking-tables-restaurants/develop

Develop

ChernyaevAV committed:
Update reservation.py

ChernyaevAV committed:
Merge pull request #361 from finding-and-booking-tables-restaurants/ChernyaevAV-patch-1

Update reservation.py

ChernyaevAV committed:
fix: удалил v1

ChernyaevAV committed:
fix: исправления для Фильтрации Броней для бизнеса + cachalot в зависимости

ChernyaevAV committed:
Merge branch 'develop' into fix/del_v1

ChernyaevAV committed:
Merge pull request #362 from finding-and-booking-tables-restaurants/fix/del_v1

Fix/del v1

ChernyaevAV committed:
Merge branch 'main' into develop

ChernyaevAV committed:
fix: рефакторинг по flake8

ChernyaevAV committed:
Merge pull request #365 from finding-and-booking-tables-restaurants/fix/del_v1

fix: рефакторинг по flake8

ChernyaevAV committed:
удален urls-v1
